src/test/java/org/jenkinsci/plugins/workflow/support/steps/ExecutorStepTest.java

```diff
@@ -527,6 +532,42 @@ void assertLogMatches(WorkflowRun build, String regexp) throws IOException { //
         });
     }
 
+    @Test public void detailsExported() throws Exception {
+        story.addStep(new Statement() {
+            @Override
+            public void evaluate() throws Throwable {
+                DumbSlave s = story.j.createOnlineSlave();
+
+                WorkflowJob p = story.j.jenkins.createProject(WorkflowJob.class, "demo");
+                p.setDefinition(new CpsFlowDefinition(
+                        "node('" + s.getNodeName() + "') {\n"
+                        + "semaphore 'wait'\n"
+                        + "    sleep 10\n"
+                        + "}"));
+
+                WorkflowRun b = p.scheduleBuild2(0).waitForStart();
+                SemaphoreStep.waitForStart("wait/1", b);
+                JenkinsRule.WebClient wc = story.j.createWebClient();
+                Page page = wc
+                        .goTo("computer/" + s.getNodeName()
+                                + "/api/json?tree=executors[currentExecutable[number,displayName,url,timestamp]]", "application/json");
+
+                JSONObject propertiesJSON = (JSONObject) (new JsonSlurper()).parseText(page.getWebResponse().getContentAsString());
+                JSONArray executors = propertiesJSON.getJSONArray("executors");
+                JSONObject executor = executors.getJSONObject(0);
+                JSONObject currentExecutable = executor.getJSONObject("currentExecutable");
+
+                assertEquals(1, currentExecutable.get("number"));
+
+                assertEquals("part of " + p.getName() + " #1",
```

Comment:
nit: It looks like this message is actually [constructed using `WorkflowRun#getFullDisplayName`](https://github.com/jenkinsci/workflow-durable-task-step-plugin/blob/5224a7b917127a66d6a14980f1ff98c614af2a6d/src/main/java/org/jenkinsci/plugins/workflow/support/steps/ExecutorStepExecution.java#L471), so it would be better to use `"part of " + b.getFullDisplayName()`, although it shouldn't actually change the text in this case.

Reply:
should getName and also getFullDisplayName be tested? or just getFullDisplayName?

Reply:
Did you mean to respond to https://github.com/jenkinsci/workflow-durable-task-step-plugin/pull/68/files#r214370377? My thought there was just that it would be good to test all of the methods that you are adding `@Exported` to. You already test most of them: `tree=executors[currentExecutable[number,displayName,url,timestamp]`, but it would be good to add `fullDisplayName` to the tree parameter as well.

My point here is just that the text returned by `currentExecutable.get("displayName"))` would not match `"part of " + p.getName() + " #1"` if you changed the display name of the WorkflowRun, or placed the WorkflowJob in a folder, or if it wasn't the first build, etc., so using `"part of " + b.getFullDisplayName()` instead makes the test slightly more robust.